rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt

```diff
@@ -1281,6 +1297,7 @@ class InternalInterpreter(val systemInterface: SystemInterface) {
     }
 }
 
+
```

Comment:
I think we should remove this extra blank line

Reply:
done